### DISCUSSION TITLE
Welcome to Gordian Wallet & Server Discussions!

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Gordian#73] [General Discussion]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fonta1n3** [comment]
Hello,

Great idea @ChristopherA 

I am the lead dev for both projects and actively working on them, currently finishing up adding the ability for Gordian-Server to install c-lightning then will be moving back to focus on Gordian-Wallet.

**Comment by cuongleqq** [comment]
Hi, everyone.

I manage engineering resources & process at Bitmark company. Currently we are focusing our efforts on making the Signer App and maybe the Gordian Wallet on Android.

**Comment by wolfmcnally** [comment]
Hi all. I'm [Wolf McNally](https://wolfmcnally.com). I've created a number of [research papers/proposals](https://github.com/blockchaincommons/research) for Blockchain Commons, including [Uniform Resources (URs)](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md) and written a number of reference implementations for those proposals. I'm also on [Twitter](https://twitter.com/wolfmcnally).

**Comment by henkvancann** [comment]
Hi all, I am [Henk](https://github.com/henkvancann). I am founder of [Blockchainbird](https://github.com/blockchainbird) (Not-for-profit, open source, etc), which is also patron of Blockchain Commons since July 2020.
Our darling is https://blockchainbird.com/t/btc/, who simply does not want to fly yet, but we love her and nurture her endlessly. 
Commercial activities, if any because customers tend to hold you back from important work ;), are done via BCWS.io.
